client/common/cmdline.c

```diff
@@ -2903,29 +2990,24 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 		}
 		CommandLineSwitchCase(arg, "app")
 		{
-			if (!freerdp_settings_set_string(settings, FreeRDP_RemoteApplicationProgram,
-			                                 arg->Value))
+			int rc = parse_app_options(settings, arg);
+			if (rc != 0)
+				return rc;
+		}
+		CommandLineSwitchCase(arg, "load-balance-info")
+		{
+			if (!copy_value(arg->Value, (char**)&settings->LoadBalanceInfo))
 				return COMMAND_LINE_ERROR_MEMORY;
 
-			settings->RemoteApplicationMode = TRUE;
-			settings->RemoteAppLanguageBarSupported = TRUE;
-			settings->Workarea = TRUE;
-			settings->DisableWallpaper = TRUE;
-			settings->DisableFullWindowDrag = TRUE;
+			settings->LoadBalanceInfoLength = (UINT32)strlen((char*)settings->LoadBalanceInfo);
 		}
+#if defined(WITH_FREERDP_DEPRECATED_COMMANDLINE_COMMANDLINE)
```

Comment:
`WITH_FREERDP_DEPRECATED_COMMANDLINE_COMMANDLINE` -> `WITH_FREERDP_DEPRECATED_COMMANDLINE`

Reply:
hmm, can not see this with grep? maybe a github artifact (did fix this with the last force push 5 days earlier?)

Reply:
> hmm, can not see this with grep? maybe a github artifact (did fix this with the last force push 5 days earlier?)

No, during review, I go from commit to commit. So, might be that you changed that in a follow-up commit. So, the overall change won't include that with a grep search.